In [20]:
n=int(input('Enter total student number:'))
m=0

for i in range(n):
    stid=input('Enter student id:')
    
    for j in str(stid):
        m+=int(j)
if m>=5 and m<=10:
    print('Ciao',end='')
elif m>=11 and m<=15:
    print('Hello',end='')
elif m>=20 and m<=25:
    print('Hola',end='')
elif m>=30 and m<=35:
    print('Salut',end='')
else:
    print('ByeBye',end='')

Enter total student number:1
Enter student id:20202020
Ciao

𝗤𝘂𝗲𝘀𝘁𝗶𝗼𝗻:
Write a python program that takes N student ids as input from the user (N 
is an input). Then calculates the sum of digits for each student id. 
Finally, print the statements from the following chart.
If the digit total is from 5 to 10 (inclusive), then print “Ciao”.
If the digit total is from 11 to 15(inclusive), then print “Hello”.
If the digit total is from 20 to 25(inclusive), then print “Hola”.
If the digit total is from 30 to 35(inclusive), then print “Salut”.
For any other valid number, print “ByeBye”.

Assume, the user will give valid student ids with 8 digits. You are not 
allowed to use lists for this task.
================================================
Sample Input 1:
2
21101710
15929010

Sample Output 1:
Hello
ByeBye

Explanation 1:
For 21101710, the total of the digits is 13 which falls in the range of 11 
to 15, so it prints “Hello”.
For 15929010, the total of the digits is 27 which does not fall in any 
specified range, so it prints  “ByeBye”.
================================================

Sample Input2:
4
20202020
10909010
17030050
13535059

Sample Output2:
Ciao
Hola
ByeBye
Salut

Explanation2:
For 20202020, the sum of the digits is 8 which falls in the range of 5 to 
10, so it prints “Ciao”.
For 10909010, the sum of the digits is 20 which falls in the range of 20 to 
25, so it prints “Hola”.
For 17030050, the sum of the digits is 16 which does not fall in any 
specified range, so it prints “ByeBye”.
For 13535059, the sum of the digits is 31 which falls in the range of 30 to 
35, so it prints “Salut”.